### ISSUE TITLE
API Pagination redirecting 

### ISSUE THREAD

**Issue by theaaron123** [novaramedia/novaramedia-com#126]
https://novaramedia.com/api/articles/2 redirects to https://novaramedia.com/api/articles/ with "page" : 1

**Comment by varffs** [comment]
Yes the custom endpoints are definitely now discontinued. We internally don't used pagination on them for anything. In order to get properly paginated articles posts make a request with the WP API posts endpoint and a category argument: https://developer.wordpress.org/rest-api/reference/posts/#arguments

We are unlikely to change top level categories for articles, audio, video any time soon so you will probably be safe using the ID hardcoded but obviously could look it up each time if preferred.

**Comment by varffs** [comment]
Discontinued
